### PR TITLE
LPS-120562 Test DefaultSegmentsEntryProvider#getSegmentsEntryIds when the user is created after the segment

### PR DESCRIPTION
**Pull request by darquesdev**
Add integration test for the default SegmentsEntryProvider#getSegmentsEntryIds method covering teh scenario when the model (user in this case) is created after the segment

### CODE REVIEW

**Comment by darquesdev** [comment]
ci:test:sf

**Comment by darquesdev** [comment]
ci:test:relevant

File: modules/apps/segments/segments-test/src/testIntegration/java/com/liferay/segments/provider/test/DefaultSegmentsEntryProviderTest.java [hunk 1]
```diff
@@ -444,6 +446,39 @@ public void testGetSegmentsEntryIdsWithSingleModelCriterion()
...
+
+		_userSegmentsCriteriaContributor.contribute(
+			criteria, String.format("(firstName eq '%s')", firstName),
+			Criteria.Conjunction.AND);
+
+		SegmentsEntry segmentsEntry = SegmentsTestUtil.addSegmentsEntry(
+			_group.getGroupId(), CriteriaSerializer.serialize(criteria),
+			User.class.getName());
+
+		_user1 = UserTestUtil.addUser();
```
Comment (cgoncas) [inline modules/apps/segments/segments-test/src/testIntegration/java/com/liferay/segments/provider/test/DefaultSegmentsEntryProviderTest.java:465]: could we create the user with the right name in only one step?

Comment (darquesdev) [reply modules/apps/segments/segments-test/src/testIntegration/java/com/liferay/segments/provider/test/DefaultSegmentsEntryProviderTest.java:465]: Good point, improving the user creating..

**Comment by darquesdev** [comment]
ci:stop

**Review by cgoncas — APPROVED** [review]
LGTM 🥇

**Comment by darquesdev** [comment]
ci:test:relevant

**Comment by darquesdev** [comment]
ci:test:sf

**Comment by cgoncas** [comment]
ci:forward

**Pull request closed without merging**